src/components/Header/Form/Form.js

```diff
@@ -0,0 +1,135 @@
+import React, { useState } from 'react';
+import { Button } from '../../Button';
+import { Search } from '../../../Icons';
+import { dataAvailable } from './config';
+
+export function Form({ setResults }) {
+  const [search, setSearch] = useState('');
+
+  function hotelSearch(event) {
+    // const searchValue = event.target.value;
+    setSearch(event.target.value);
+    setResults(
+      dataAvailable.filter(
+        (hotel) =>
+          hotel.name &&
+          search &&
+          hotel.name.toLowerCase().includes(event.target.value.toLowerCase()),
```

Comment:
Pass search instead of event.target.value. And Now your search works only with hotel name